digitalpalireader/content/js/send_sidebar.js

```diff
@@ -588,10 +588,13 @@ var DPRSend = {
 				oldTabBrowser.contentDocument.getElementById('dpr-search-browser').contentWindow.searchTipitaka(which,getstring,MAT,sets,book,part,rx);
 			}
 		}
-		else {
+		else if (DPR_PAL.isXUL) {
 			var permalink = 'chrome://digitalpalireader/content/search.xul' + '?type='+which+'&query=' + getstring + '&MAT=' + MAT + '&set=' + sets + '&book=' + book + '&part=' + part + '&rx=' + rx;
 			DPRChrome.openDPRTab(permalink,'DPRs');
-		}
+		} else {
+      const url = DPR_PAL.dprHomePage + '?feature=search&type='+which+'&query=' + getstring + '&MAT=' + MAT + '&set=' + sets + '&book=' + book + '&part=' + part + '&rx=' + rx;
```

Comment:
since this is new code can we use the es6 template literals to make it clearer?

Reply:
Sure, will try to do it.